backend/tracim_backend/models/data.py

```diff
@@ -716,11 +723,17 @@ def copy(
 
         # copy attached_file
         if revision.depot_file:
-            copy_rev.depot_file = FileIntent(
-                revision.depot_file.file.read(),
-                revision.file_name,
-                revision.file_mimetype,
-            )
+            try:
+                copy_rev.depot_file = FileIntent(
+                    revision.depot_file.file.read(),
+                    revision.file_name,
+                    revision.file_mimetype,
+                )
+            except IOError as exc:
+                raise NewRevisionAbortedDepotCorrupted(
```

Comment:
`NewRevisionAbortedDepotCorrupted` => `CopyRevisionAbortedDepotCorrupted` (this allow to distinguish the two cases)

Reply:
done